### PR TITLE
[Issue 7] Update import torchvision.models as models

### PR DESCRIPTION
**Pull request by gesuwen**
Create a new _UpdateTorchvisionModelsImports class to replace all "import torchvision.models as models" to "from torchvision import models"

### CODE REVIEW

**Comment by gesuwen** [comment]
@kit1980 I am aware that we could refactor the new class I added and _UpdateFunctorchImports into a generalized class for import replacement. Shall I do it here, or leave it for a future commit?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .pytest_cache/
 __pycache__/
 *.egg-info/*
+torchfix/visitors/deprecated_symbols/__init__.py
```
Comment (kit1980) [inline .gitignore:5]: Remove this?

File: tests/fixtures/deprecated_symbols/codemod/torchvision_models.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import torch
+import torchvision.models as models
+
```
[the remark this answers is not in the document]
Comment (kit1980) [reply tests/fixtures/deprecated_symbols/codemod/torchvision_models.py:3]: Test `from torchvision.models import resnet50, resnet101`: should leave as is.

[the remark this answers is not in the document]
Comment (kit1980) [reply tests/fixtures/deprecated_symbols/codemod/torchvision_models.py:3]: Test `import torchvision.models`

**Comment by kit1980** [comment]
@gesuwen Thank you for the PR.

I've left some comments and I think in general this should be a separate rule with a separate code, like TOR201, to report the issue.
Just reporting the issue should be probably done first.
Changing the code can be done later, in a different PR even.

File: .gitignore (hunk 1 above)
Comment (gesuwen) [reply .gitignore:5]: Removed in commit 2

File: tests/fixtures/deprecated_symbols/codemod/torchvision_models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (gesuwen) [reply tests/fixtures/deprecated_symbols/codemod/torchvision_models.py]: Updated the new test file with all changes suggested in commit 2

**Comment by gesuwen** [comment]
> @gesuwen Thank you for the PR.
> 
> I've left some comments and I think in general this should be a separate rule with a separate code, like TOR201, to report the issue. Just reporting the issue should be probably done first. Changing the code can be done later, in a different PR even.

@kit1980 Thanks for your review! I just updated the PR, resolving all suggestions. Commit 2 move the new visitor to the vision dir and include giving LintViolation warning + replacement option. Could you please help review again and let me know if there is any addition suggestion?

**Review by kit1980 — APPROVED** [review]

**Comment by kit1980** [comment]
@gesuwen Looks good to me, thanks!

I'll do the codemod part later after factoring some things around import replacements.

**Pull request merged**